### ISSUE TITLE
Make faq-extension point to FAQ source file

### ISSUE THREAD

**Issue by hfuhruhurr** [jupyterlab/jupyterlab#4574]
In #3835, @ian-r-rose [made some suggestions](https://github.com/jupyterlab/jupyterlab/issues/3835#issuecomment-388908756) for a path forward.  My PR #4573 addressed items 1 and 2 (converted, moved, proofed FAQ).  This issue is to address items 3 and 4.

@ellisonbg @ivanov

**Comment by ivanov** [comment]
Inlining @ian-r-rose's  suggestions here so I can respond to them:

> 3. Add the ability to open the FAQ page in readthedocs to the @jupyterlab/help-extension, probably using an iframe. This would be handled similarly to how the help links are set up in that same extension.
> 4. Delete (!) the @jupyterlab/faq-extension entirely.

I agree that we should not have the FAQ in two separate places. However, I do not think we should use an iframe to point to read the docs because in that case reading the FAQ would require an internet connection. Or at least we should consider that ramification as undesirable and mitigate it when we get the chance.

**Comment by ian-r-rose** [comment]
I don't think it would be too bad to have the FAQ be iframed. Currently all of the other help links (including the JupyterLab reference itself) do that, so we already require an internet connection to get to that information. And the FAQ itself is mostly a collection of links to other places in the docs, again requiring the internet.
